src/json_reporter.cc

```diff
@@ -78,7 +78,12 @@ bool JSONReporter::ReportContext(const Context& context) {
   out << indent << FormatKV("date", walltime_value) << ",\n";
 
   if (Context::executable_name) {
-    out << indent << FormatKV("executable", Context::executable_name) << ",\n";
+    // windows uses backslash for its path separator,
+    // which must be escaped in JSON otherwise it blows up conforming JSON
+    // decoders
+    std::string executable_name = Context::executable_name;
+    ReplaceAll(&executable_name, "\\", "\\\\");
```

Comment:
is this only ever going to be a problem for executable name or should we do this for all FormatKV where value is string?

Reply:
In principle, all json strings should be properly escaped.

Reply:
I suspect when the proper json support is here, possibly it will be able to handle all this transparently.

Reply:
This is true, but I think it doesn't matter for everything else since we know we're not dumping out escape-necessary characters in any other place.

Of course, @LebedevRI is right: proper json support with some library would likely take care of it for us.

Reply:
Since we support custom statistics, and custom counters,
it is totally possible that users could be passing strings that need escaping,
but i think it can be done separately later.